### PR TITLE
Csv

### PR DESCRIPTION
**Pull request by EHale64**
Created a Game class and tests

### CODE REVIEW

File: lib/game.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+require 'csv'
+class Game
+  attr_reader :game_id,
```
Comment (jpc20) [inline lib/game.rb:3]: We could move these attr_readers down below the class methods.

File: lib/game.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+              :away_team_id,
+              :home_team_id,
+              :away_goals,
+              :home_goals,
+              :venue,
+              :venue_link
+  def self.from_csv(file_path)
+    csv = CSV.read("#{file_path}", headers: true, header_converters: :symbol)
+
+    @@all = csv.map do |row|
```
Comment (arpariseau) [inline lib/game.rb:16]: Maybe @@all_games for consistency between our classes?

File: test/game_test.rb [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    assert_equal 30 , @game.away_team_id
+    assert_equal  52 , @game.home_team_id
+    assert_equal   2 , @game.away_goals
+    assert_equal   3 , @game.home_goals
+    assert_equal   'Providence Park' , @game.venue
+    assert_equal   '/api/v1/venues/null' , @game.venue_link
+  end
+
+  def def_test_it_can_read_games_from_CSV
+    skip
```
Comment (Ashkanthegreat) [inline test/game_test.rb]: unskip this

**Review by jpc20 — APPROVED** [review]
Nice work Eric.

File: test/game_test.rb [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    away_team_id: 30,
+    home_team_id: 52,
+    away_goals: 2,
+    home_goals: 3,
+    venue: 'Providence Park',
+    venue_link: '/api/v1/venues/null'})
+
+
+    Game.from_csv("./test/fixtures/games_fixture.csv")
+    @game_2 = Game.all[-1]
```
Comment (arpariseau) [inline test/game_test.rb:21]: Maybe @game1 and @game2 or @game and @csv_game?

**Review by arpariseau — CHANGES_REQUESTED** [review]
Gonna unskip and make sure the test works, and we'll be good

File: test/game_test.rb [hunk 5]
```diff
@@ -38,8 +38,7 @@ def test_it_has_attributes
     assert_equal   '/api/v1/venues/null' , @game.venue_link
   end
 
-  def def_test_it_can_read_games_from_CSV
```
Comment (arpariseau) [inline test/game_test.rb]: Caused this not to ever run because it's def def_test_(stuff), not def test_(stuff)

File: test/game_test.rb [hunk 6]
```diff
@@ -48,7 +47,7 @@ def def_test_it_can_read_games_from_CSV
      assert_equal 2 , @game_2.home_team_id
      assert_equal 3 , @game_2.away_goals
      assert_equal 0 , @game_2.home_goals
-     assert_equal 'Centruy Link Field' , @game_2.game_id
-     assert_equal '/api/v1/venues/null',  @game_2.game_id
+     assert_equal 'Centruy Link Field' , @game_2.venue
```
Comment (arpariseau) [inline test/game_test.rb:50]: Because it never ran, these two were never fixed from their copy-paste

**Pull request merged**